code/game/objects/random/biomes/biome.dm

```diff
@@ -0,0 +1,80 @@
+GLOBAL_LIST_EMPTY(loot_biomes)
+
+/obj/landmark/loot_biomes
+	name = "debug biome"
+	icon_state = "player-blue-cluster"
+	var/prob_secondary_tags = 33
+	var/list/main_tags = list()
+	var/list/secondary_tags = list()
+	var/list/tags_to_spawn = list()
+	var/cap_price = 500000
+	var/top_price = 10000 //top_price = cap_price / 10
```

Comment:
cap_price / 10 would equal 50000

Reply:
this variable is automatically set to the variable "cap price / 10" when the object is created.